bftengine/src/bftengine/SigManager.cpp

```diff
@@ -162,6 +167,11 @@ SigManager::SigManager(PrincipalId myId,
       LOG_DEBUG(KEY_EX_LOG, "Adding key of client " << p.first << " key size " << key.size());
     }
   }
+  if (std::get<0>(operatorKey) > 0 && !std::get<1>(operatorKey).empty()) {
```

Comment:
Use structured bindings to extract the fields out of the tuple. 
Example:
https://www.geeksforgeeks.org/structured-binding-c/